thingif/src/test/java/com/kii/thingif/SmallTestBase.java

```diff
@@ -47,4 +51,21 @@ protected void assertJSONArray(JSONArray expected, JSONArray actual) throws JSON
             }
         }
     }
+
+    protected void assertSameCommands(Command expected, Command actual) {
+        Gson gson = new Gson();
+        Assert.assertEquals(gson.toJson(expected), gson.toJson(actual));
+
+        Assert.assertTrue(Arrays.equals(
+                expected.getAliasActions().toArray(),
+                actual.getAliasActions().toArray()));
+        Assert.assertFalse(
+                expected.getAliasActionResults() != null ^
+                actual.getAliasActionResults() != null );
```

Comment:
This assertion is little bit difficult for me. How about following?

```java
Assert.assertTrue(
    expected.getAliasActionResults() != null && actual.getAliasActionResults() != null ||
    expected.getAliasActionResults() == null && actual.getAliasActionResults() == null)
```

Your code is very smart. My code is dull but I think it is more clear what we want to do.